gpkit/nomials/data.py

```diff
@@ -54,13 +50,6 @@ def __hash__(self):
                                          [str(self.units)]))
         return self._hashvalue
 
-    @classmethod
-    def fromnomials(cls, nomials):
-        """Construct a NomialData from an iterable of Signomial objects"""
-        nd = cls()  # use pass-through version of __init__
-        nd.init_from_nomials(nomials)
-        return nd
```

Comment:
I can't recall why we wrote `fromnomials`. Did we ever need it, or did we just think it might be nice to have someday?